src/app/_components/BlockList/LayoutA/__tests__/BlockListWithControls.test.tsx

```diff
@@ -4,35 +4,37 @@ import { UIBlock, UIBlockType } from '../../types';
 import { BlockListWithControls } from '../BlockListWithControls';
 import { BlockListProvider } from '../Provider';
 
+const date = new Date('2024-01-01T00:00:00Z').getTime();
```

Comment:
should fix snapshot mismatch going forward

Reply:
great